addons/test_mail_full/tests/test_rating.py

```diff
@@ -159,20 +173,35 @@ def test_rating_last_value_perfs(self):
                 record.rating_apply(5, token=access_token)
             self.flush_tracking()
 
-        with self.assertQueryCount(employee=1):
-            record_ratings._compute_rating_last_value()
-            vals = [val == 5 for val in record_ratings.mapped('rating_last_value')]
-            self.assertTrue(all(vals), "The last rating is kept.")
+        if is_rating_mixin_test:
+            with self.assertQueryCount(employee=1):
+                record_ratings._compute_rating_last_value()
+                vals = [val == 5 for val in record_ratings.mapped('rating_last_value')]
```

Comment:
we can keep just a ganarator
```suggestion
                vals = (val == 5 for val in record_ratings.mapped('rating_last_value'))
```

Reply:
yes, also better for performance, thanks